material-ui/material-ui.scss

```diff
@@ -120,3 +121,120 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  input {
+    position: relative;
+    left: 30px;
+    width: 40px;
+    height: 40px;
+    margin: 0;
+    padding: 20px;
+    transform: scale(1);
+    transition: opacity .3s, transform .2s;
+    border-radius: 50%;
+    outline: none;
+    background-color: $material-ui-blue;
+    box-shadow: none;
+    opacity: 0;
+    z-index: -1;
+    appearance: none;
+    pointer-events: none;
+
+    &:checked,
+    &:indeterminate {
+      background-color: $material-ui-blue;
+    }
+
+    &:checked + span {
+      &::before {
+        border-color: $material-ui-blue;
+        background-color: $material-ui-blue;
+      }
+
+      &::after {
+        border-color: $white;
+      }
+    }
+
+    &:indeterminate + span {
+      &::before {
+        border-color: $material-ui-blue;
+        background-color: $material-ui-blue;
+      }
+
+      &::after {
+        transform: translate(4px, 3px);
+        border-left: 0;
+        border-color: $material-ui-blue;
+      }
+    }
+
+    &:active {
+      transform: scale(0);
+      transition: transform 0s, opacity 0s;
+      opacity: 1;
+
+      + span {
+        &::before {
+          border-color: $material-ui-blue;
+        }
+      }
+    }
+
+    &:checked {
+      &:active + span {
+        &::before {
+          border-color: transparent;
+          background-color: $material-ui-blue;
+        }
+      }
+    }
+
+    &:focus {
+      opacity: .12;
+    }
+  }
+
+  span {
+    &::before {
+      position: relative;
+      top: 1px;
+      left: -2px;
+      width: 18px;
+      height: 18px;
+      float: left;
+      transition: border-color .2s, background-color .2s;
+      border: solid 2px;
+      border-radius: 2px;
+      border-color: $material-ui-disabled;
+      content: '';
+      vertical-align: top;
+      box-sizing: border-box;
+    }
+
+    &::after {
+      position: relative;
+      left: -20px;
+      width: 10px;
+      height: 5px;
+      float: left;
+      transform: translate(3px, 4px) rotate(-45deg);
+      border: solid 2px transparent;
```

Comment:
See comments above about using a SCSS variable to calculate all the above dimensions.